cli/print/clusters.go

```diff
@@ -15,19 +16,41 @@ var clustersTmplSrc = `ID	NAME	K8S DISTRO	K8S VERSION	STATUS	CREATED	EXPIRES
 
 var clustersTmpl = template.Must(template.New("clusters").Funcs(funcs).Parse(clustersTmplSrc))
 
-func Clusters(w *tabwriter.Writer, clusters []*types.Cluster) error {
-	if err := clustersTmpl.Execute(w, clusters); err != nil {
+func Clusters(outputFormat string, w *tabwriter.Writer, clusters []*types.Cluster) error {
+	if outputFormat == "table" {
+		if err := clustersTmpl.Execute(w, clusters); err != nil {
+			return err
+		}
+		return w.Flush()
+
+	}
+	if outputFormat == "json" {
+		defer w.Flush()
+		var cAsByte []byte
+		if len(clusters) == 1 {
+			cAsByte, _ = json.MarshalIndent(clusters[0], "", "  ")
+		} else {
+			cAsByte, _ = json.MarshalIndent(clusters, "", "  ")
+		}
```

Comment:
Same here - changing the outer type in the JSON makes composable scripts harder to write.